### DISCUSSION TITLE
Step 2:  Code with Github Copilot

### DISCUSSION THREAD

**Discussion by T095T** [skills/.github#296] [Code with Copilot]
After following all the required steps, I am not able to proceed further.
I have created codespace for main branch and created skills.js file, but after entering all commands in terminal and refreshing after 60 seconds
I am not able to proceed to step .

**Comment by T095T** [comment]
@sinsukehlab Can u help me here?

**Reply by sinsukehlab** [reply]
You have made two mistakes.
1. You should have named the file "skills.js", not "skill.js", in order to trigger the workflow.
1. The function name should be "calculateNumbers" (case-sensitive). The workflow will grep the file "skills.js" for the string "function calculateNumbers".

**Reply by sinsukehlab** [reply]
@T095T It seems that you were able to finish the course. Please mark my reply as the answer if you think it helped you.
